consensus/config.go

```diff
@@ -11,13 +11,15 @@ const (
 	viewChangeDuration time.Duration = 30 * time.Second
 
 	// timeout duration for announce/prepare/commit
-	phaseDuration time.Duration = 20 * time.Second
+	phaseDuration time.Duration = 60 * time.Second
 	maxLogSize    uint32        = 1000
 )
 
+type timeoutType int
+
 const (
-	timeoutNormal     = "normal"
```

Comment:
I think change "Normal" to "Consensus" will be more informative.

Reply:
OK